### DISCUSSION TITLE
How to change bot console text?

### DISCUSSION THREAD

**Discussion by GameHackerTV** [Cog-Creators/Red-DiscordBot#5704] [Q&A] [answered]
Good afternoon, how to change the output of data in the console?
![IMG_20220518_141944](https://user-images.githubusercontent.com/86518375/169057769-dac1ab0f-df72-409c-927a-720ec570a0fc.jpg)
For example, change the color of the Red-DiscordBot inscription, or even change it to something.

and I would like to change the LINK to the bot invitation (in the console).
Where can I find this data to change it.
Help pls 🙏

**Comment by Kowlin** [comment, marked as answer]
Considering a bot owner should only be seeing this information for the most part, this is not something we're willing to open modifications for it.

**Reply by GameHackerTV** [reply]
ah, another question. Can I remove the inscription Red v3? in help embed
so that it was: Bot Help Menu and then there were commands, by default
 Bot Help Menu
Red v3 
 then commands.

**Reply by yamikaitou** [reply]
`[p]set description`
